Zend/tests/readonly_classes/readonly_class_missing_type2.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Promoted properties of a readonly class must have type
+--FILE--
+<?php
+
+readonly class Foo
+{
+    public function __construct(
+        private readonly $bar
```

Comment:
Should omit `readonly` here to make sure it tests the right thing?

Reply:
Yeah, you are right